easybuild/easyblocks/s/siesta.py

```diff
@@ -0,0 +1,308 @@
+                'SiestaSubroutine/FmixMD/Src/para',
+                'SiestaSubroutine/FmixMD/Src/simple',
+                'STM/simple-stm/plstm', 'STM/ol-stm/Src/stm',
+                'VCA/mixps', 'VCA/fractional',
+                'Vibra/Src/vibra', 'Vibra/Src/fcbuild',
+                'WFS/info_wfsx', 'WFS/wfsx2wfs',
+                'WFS/readwfx', 'WFS/wfsnc2wfsx', 'WFS/readwf', 'WFS/wfs2wfsx',
+            ]
+
+            if LooseVersion(self.version) <= LooseVersion("4.0"):
+                expected_utils.extend([
+                    'Bands/new.gnubands',
+                    'TBTrans/tbtrans',
+                ])
+
+            if LooseVersion(self.version) >= LooseVersion("4.0"):
+                expected_utils.extend([
+                    'SiestaSubroutine/ProtoNEB/Src/protoNEB',
+                    'SiestaSubroutine/SimpleTest/Src/simple_pipes_parallel',
+                    'SiestaSubroutine/SimpleTest/Src/simple_pipes_serial',
+                    'Sockets/f2fmaster', 'Sockets/f2fslave',
+                ])
+
+            if LooseVersion(self.version) >= LooseVersion("4.1"):
+                expected_utils.extend([
+                    'Bands/gnubands',
+                    'Grimme/fdf2grimme',
+                    'SpPivot/pvtsp',
+                    'TS/ts2ts/ts2ts', 'TS/tshs2tshs/tshs2tshs', 'TS/TBtrans/tbtrans',
+                ])
+
+            for f in expected_utils:
+                shutil.copy(os.path.join(self.cfg['start_dir'], 'Util', f), bindir)
+
+        if self.cfg['with_transiesta']:
+            # Build transiesta
+            try:
+                os.chdir(obj_dir)
+            except OSError, err:
+                raise EasyBuildError("Failed to change back to Obj dir: %s", err)
+
+            run_cmd('make clean', log_all=True, simple=True, log_output=True)
+            run_cmd('make -j %s transiesta' % self.cfg['parallel'], log_all=True, simple=True, log_output=True)
+
+            shutil.copy(os.path.join(self.cfg['start_dir'], 'Obj', 'transiesta'),
+                        bindir)
+
+    def build_step(self):
+        """No build step for Siesta."""
+        pass
+
+    def install_step(self):
+        """Custom install procedure for Siesta."""
+
+        try:
+            # binary
+            bindir = os.path.join(self.installdir, 'bin')
+            shutil.copytree(os.path.join(self.cfg['start_dir'], 'bin'), bindir)
+
+        except OSError, err:
+            raise EasyBuildError("Failed to install Siesta: %s", err)
+
+    def sanity_check_step(self):
+        """Custom sanity check for Siesta."""
+
+        bins = ['bin/siesta']
+
+        if self.cfg['with_transiesta']:
+            bins.extend(['bin/transiesta'])
+
+        if self.cfg['with_utils']:
+            bins.extend(['bin/denchar'])
+
+        custom_paths = {
+            'files': bins,
+            'dirs': [],
+        }
+
+        super(EB_Siesta, self).sanity_check_step(custom_paths=custom_paths)
```

Comment:
yesterday I had only tested with an `intel` toolchain, today checked with `foss`

[this](https://gist.github.com/migueldiascosta/0fbe52345abe7d59eb62a0cbab909264#file-siesta-py-patch) is what I changed to get all versions to work on both toolchains